drivers/pwm/pwm_stm32.c

```diff
@@ -233,7 +238,9 @@ static int pwm_stm32_pin_set(const struct device *dev, uint32_t pwm,
 		oc_init.OCState = LL_TIM_OCSTATE_ENABLE;
 		oc_init.CompareValue = pulse_cycles;
 		oc_init.OCPolarity = get_polarity(flags);
+#ifndef CONFIG_SOC_SERIES_STM32L0X
```

Comment:
Could probably be removed as it is initialized by `LL_TIM_OC_StructInit` if available.

Reply:
Correct also here. Double-checked the init function for all MCUs. It is safe to be removed.